test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -18,44 +22,110 @@ public TagHelperSampleTest(MvcTestFixture<TagHelperSample.Web.Startup> fixture)
 
         public HttpClient Client { get; }
 
-        public static TheoryData<string> PathData
+        [Fact]
```

Comment:
The goal of these tests is to validate that a portion of the site is up and running. Is it really beneficial to separate this into multiple tests? As it is now it thoroughly tests one piece; especially since it's stateful.